docs/plugin-syntax-dynamic-import.md

```diff
@@ -36,3 +36,28 @@ require("@babel/core").transform("code", {
 });
 ```
 
+## Dynamic imports with babel-preset-env
+
+The babel-preset-env plugin doesn't know that the dynamic import plugin with translate `import` into a `Promise`. For example, IE requires the `promise` and `iterator` polyfill to be added manually. 
```

Comment:
>that the dynamic import plugin with translate

Typo: plugin _will_ translate

Also, these docs would be very helpful on the site!